modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -191,8 +193,11 @@ object FormInputEV {
         }
       )
 
-    val onKeyDown: ReactKeyboardEventFromInput => Callback = e =>
-      $.setStateL(State.lastKeyCode)(e.keyCode) *> clearStateCursor
+    def onKeyDown(props: Props[EV, A], state: State): ReactKeyboardEventFromInput => Callback = e =>
+      if (e.keyCode === 13)
```

Comment:
You can use `KeyCode.Enter` instead of `13`

Reply:
Thanks for the tip, I'll change to that.